src/bin.ts

```diff
@@ -69,7 +71,8 @@ const {
   '--skip-ignore': skipIgnore = DEFAULTS.skipIgnore,
   '--prefer-ts-exts': preferTsExts = DEFAULTS.preferTsExts,
   '--log-error': logError = DEFAULTS.logError,
-  '--build': build = DEFAULTS.build
+  '--build': build = DEFAULTS.build,
+  '--script-mode': scriptMode = DEFAULTS.scriptMode
```

Comment:
I’d be happy to call this `script` or even just `rel` (`relative`) since that’s all this is really doing. Script mode seems a bit expansive for the simplicity of the feature.

Reply:
yeah, wasn't happy with the name wither, but just wanted to make it work
`relative` sounds better since `script` can be ambiguous in `typescript` context :)